CHANGELOG.md

```diff
@@ -1,33 +1,68 @@
 # Change Log
 
-## [1.8.0](https://github.com/test-kitchen/test-kitchen/tree/1.8.0) (2016-05-05)
```

Comment:
There may be a better way to do this but I would just copy what is generated for 1.9.0 and paste it onto the former changelog